src/app/app.component.css

```diff
@@ -13,3 +13,5 @@
 .app-footer {
```

Comment:
![image](https://user-images.githubusercontent.com/54113846/80978743-209d2c80-8e2f-11ea-9c03-2b4c03c7c384.png)

Reply:
Please fix errors in case you don't have any results in the component's template

Reply:
![image](https://user-images.githubusercontent.com/54113846/80978967-6954e580-8e2f-11ea-94ce-f1359ffd1f7e.png)

Reply:
Please, double check mockup and your layout and add appropriate changes(margins, elements' dimensions and so on...)

Reply:
Please, also clear search results in case you've closed the search

Reply:
![image](https://user-images.githubusercontent.com/54113846/80980398-462b3580-8e31-11ea-90a8-48a9b2c2beeb.png)

Reply:
![image](https://user-images.githubusercontent.com/54113846/80980478-57744200-8e31-11ea-87e3-749f67c1bdc5.png)

Reply:
Search elements should be aligned to the screen center on the small resolutions too(as a example, please, use header or eco-news dashboard)

Reply:
fixed